_docs_integrate/attribute-introduction.md

```diff
@@ -21,3 +21,132 @@ required_by:
+
+### IdentityAttribute value types
+
+The [IdentityAttribute value types]({% link _docs_integrate/attribute-values.md %}#identity-attributes) are used to define the type of the `value` property of an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute). In order to fulfill various purposes, many such IdentityAttribute value types are provided. If the IdentityAttribute value type used is not sufficient to clearly characterize the IdentityAttribute, `tags` can be used for a more precise description of its meaning.
+
+The storage of multiple IdentityAttributes of the same value type is possible. If an Identity receives a [Request for an IdentityAttribute of a certain value type]({% link _docs_integrate/read-attributes-from-peer.md %}#example-of-reading-an-identityattribute), it can choose which of the matching IdentityAttributes it wants to provide for its [Response]({% link _docs_integrate/data-model-overview.md %}#response) to the [Request]({% link _docs_integrate/data-model-overview.md %}#request). For example, storing multiple IdentityAttributes of the same value type is beneficial if an Identity has more than one residential address. In order to be able to distinguish such IdentityAttributes from each other, it is crucial to specify appropriate `tags` for them.
+{: .notice--info}
+
+Depending on what IdentityAttribute value type is used, the associated IdentityAttribute is either a simple IdentityAttribute or a complex IdentityAttribute.
+
+#### Simple IdentityAttributes
+
+A simple IdentityAttribute is an IdentityAttribute with an [IdentityAttribute value type]({% link _docs_integrate/attribute-values.md %}#identity-attributes) for which none of its properties can be interpreted as another IdentityAttribute value type. In most cases, the IdentityAttribute value type then only has a single property, which often also has the name `value`. This property stores the actual value of the IdentityAttribute. From a technical point of view, it can be stated that the `id` of a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) whose `content` is given by a simple IdentityAttribute cannot be the `parentId` of another LocalAttribute. In other words, it could be said that simple IdentityAttributes are not composite. Examples of simple IdentityAttributes are IdentityAttributes for which [DisplayName]({% link _docs_integrate/attribute-values.md %}#displayname) or [EMailAddress]({% link _docs_integrate/attribute-values.md %}#emailaddress) is used as the IdentityAttribute value type.
```

Comment:
- I'm having trouble finding a suitable alternative, do you have any suggestions? I don't want to write that the property **has** a different IdentityAttribute value type or **corresponds** to a different IdentityAttribute value type, because this is not really the case: For example, you specify the `day` of a [BirthDate](https://enmeshed.eu/integrate/attribute-values#birthdate) as a string and not as a [BirthDay](https://enmeshed.eu/integrate/attribute-values#birthday), although the `day` property can also be understood as a BirthDay (see validation column) and therefore the corresponding RepositoryAttribute is created.
- 👍 
- 👍

Reply:
Good point! Will change it accordingly.